### DISCUSSION TITLE
Icons in Controls — Allow us to also choose from the built-in icons

### DISCUSSION THREAD

**Discussion by donn-xx** [godotengine/godot-proposals#8192] [Editor]
Not sure if this is already a thing, but I often want to use an icon on a button (or some other control) and I just want to use whatever the engine already has baked-in. 

Would it be possible to make a resource type that allows this? Like "New BuiltinEditorIcon".

![image](https://github.com/godotengine/godot-proposals/assets/392218/f142848a-bc61-432c-9287-ac008c861df1)

**Comment by Calinou** [comment]
For binary size reasons, editor icons are not included in export templates or even when you run a project in the first place, so this would result in the icon disappearing once you run the project.

**Reply by donn-xx** [reply]
But it would be useful for in-editor tools.
